inst/htmlwidgets/r3dmol.yaml

```diff
@@ -4,10 +4,10 @@ dependencies:
     src: htmlwidgets/lib/jquery-3.5.1
     script: jquery.min.js
   - name: r3dmol
-    version: 1.5.4
-    src: htmlwidgets/lib/3Dmol-1.5.4
+    version: 1.6.2
+    src: htmlwidgets/lib/3Dmol-1.6.2
     # script: 3Dmol.js            # Debug
-    # script: 3Dmol.min.js        # Production with jquery
+    # script: 3Dmol-min.js        # Production with jquery
     # script: 3Dmol-nojquery.js   # Debug without jquery
-    script: 3Dmol-nojquery.min.js # Production without jquery
+    script: 3Dmol-nojquery-min.js # Production without jquery
```

Comment:
It dosen't need to change `.` to `-` before `min`.

And it would be better to rename the directory name to 1.6.2 as one commit, and update the code in js file as another commit.

Moving files and modifying files in one commit will lose the differences, which is harder to trace the differences.

Reply:
I found that when building 1.6.2 with `gulp` it makes files with `*-min.js` instead of `*.min.js`. I had to change that script source otherwise it would no longer work because it's pointing to the wrong file.

I'll split up the change of files and change of folder into two different commits.